pallets/aggregate/src/lib.rs

```diff
@@ -781,6 +832,62 @@ pub mod pallet {
 
             Ok(owner.post_info(None))
         }
+
+        /// Set gas price for a given domain ID
+        /// Gas price is X in the following formula - 1 TOKEN in destination chain = X origin tokens
+        #[pallet::call_index(4)]
+        pub fn set_gas_price(
+            origin: OriginFor<T>,
+            domain_id: u32,
+            gas_price: BalanceOf<T>,
+        ) -> DispatchResultWithPostInfo {
```

Comment:
If you didn't need to change the weight (that's the case here), you can use a simple `DispatchResult`

Reply:
does not apply anymore, this extrinsic will me removed